File: backend/src/test/java/pl/edu/ur/blokur/BlokurApplicationTests.java
package pl.edu.ur.blokur;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

/** Testy kontekstu aplikacji Spring Boot. */
@Disabled("Tymczasowo wyłączone - brak skonfigurowanej bazy danych dla testów")
@SpringBootTest
class BlokurApplicationTests {

    @Test
    void contextLoads() {}
}

File: backend/src/test/kotlin/pl/edu/ur/blokur/BlokurApplicationTests.kt
package pl.edu.ur.blokur

import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest

@Disabled("Tymczasowo wyłączone - brak skonfigurowanej bazy danych dla testów")
@SpringBootTest
class BlokurApplicationTests {
    @Test
    fun contextLoads() {
    }
}
